src/components/AssignmentSummary.jsx

```diff
@@ -130,6 +130,15 @@ class AssignmentSummary extends Component {
               contactsFilter: 'reply',
               hideIfZero: true
             })}
+            { assignment.campaign.useDynamicAssignment ? this.renderBadgedButton({
```

Comment:
I wonder if this option should be tied to bulk send instead of dynamic assignment.

The tricky thing with this feature for non-bulk sending, is that the texter is basically mashing the Enter key down as quickly as the UI will respond.  This means that if it suddenly switches to replies they won't necessarily catch it and maybe send a default script value that's not appropriate.

If that makes sense to you, then let's condition it on `window.ALLOW_SEND_ALL`